src/DisplayBoard.cpp

```diff
@@ -0,0 +1,110 @@
+#include "DisplayBoard.hpp"
+
+namespace
+{
+    using u_int = unsigned int;
+
+    const char HORIZONTAL = '-';
+    const char VERTICAL = '|';
+    const char INTERSECTION = '+';
+    const char PADDING = ' ';
+
+
+    std::string drawPadding(const u_int width, const char sign = PADDING);
+    std::string drawEndLine(const u_int width);
+    std::string drawEmptyLine(const u_int width, const char sign = PADDING);
+    std::string drawColumns(const u_int maxElementsInRows, const u_int colNumber, const u_int heightCol);
+    std::string drawRow(const u_int maxElementsInRows);
+    std::stringstream drawBoard(const u_int width,
+                                const u_int height,
+                                const u_int widthRows,
+                                const u_int heightCol);
+}
+
+namespace DisplayBoard
+{
+    std::string displayInterface(const Board& b){
+        auto width = b.getRowsNumber();
+        auto height = b.getColsNumber();
+        auto widthRows = b.findLongestVectorInRows();
+        auto heightCol = b.findLongestVectorInCols();
+
+        auto tmp = drawBoard(width, height, widthRows, heightCol);
+        auto tmpStr = tmp.str();
+        std::cout << tmpStr << "\n";
+
+        return tmpStr;
+    }
+}
+
+namespace
+{
+    std::stringstream drawBoard(const u_int width,
+                                const u_int height,
+                                const u_int widthRows,
+                                const u_int heightCol)
+    {
+        std::stringstream ss;
+        std::string s = drawEndLine(widthRows);
+        s.pop_back();
+
+        ss << drawColumns(widthRows, width, heightCol);
+        ss << s << drawEndLine(width) << "\n";
+        for (int i = height;i > 0 ; --i)
```

Comment:
space after first ;

Reply:
thanks :)

Reply:
Why do you iterate backward? Normally we do it forward if there is no difference like here,

Reply:
@majkel84 @OriaTori 
could you tell something about it?

Reply:
I don't know why it was written like this but I changed this condition in this c870f16 commit.